operations.yaml

```diff
@@ -63,5 +66,6 @@ inverseOperationIds:
   - removeEnvironment
   - getEnvironment
   - getAllEnvironments
-  - toggleEnvironmentOff
```

Comment:
Don't need this, superceeded by AddEnvironmentToProject/RemoveEnvironmentFromProject